### ISSUE TITLE
Multiplayer suddenly not working

### ISSUE THREAD

**Issue by xnnox70** [UltimMC/Launcher#227]
### Operating System

Windows 11

### Description of bug

Tried joining on a server after days of not playing. I try to join, but there's an error "Invalid signature for public profile key" shows up, and tells me to restart the game. I tried logging off and on, reinstalling the client to no avail. I thought this wasn't the launcher's issue, however, this didn't happen on another launcher I used.

### Steps to reproduce

• Run instance
• Multiplayer>>join server>>connect
• "Invalid signature for public profile key"

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by Cyimil** [comment]
It seems that since 1.19.1 was launched, UltimMC started having this problem...

**Comment by MenacingPerson** [comment]
From authlib injector notes:

```
This update contains breaking changes that require both server and client to use v1.2.0 and above of authlib-injector!

If you encounter the Invalid signature for profile public key. Try restarting your game. error when entering the server, please make sure that the authlib-injector version of the server and client is v1.2.0 or above.

If your server version is 1.19 or above, please modify server.properties and modify enforce-secure-profile to true!

     [Fix] Fixed a warning that chat messages were not signed in MC 1.19.1

Linked pull request: #172
Build number: 48
```

I think ultimmc devs have to fix

**Comment by jwaxy** [comment]
I have the same error with Linux.

**Comment by jwaxy** [comment]
I fixed the problem with using Wurst Client's nochatreports and noencryption modules.

**Comment by sedub01** [comment]
Yeah, I have that problem too but people around me say there's no errors
Hope devs will fix this asap

**Comment by hiarthurbr** [comment]
I was having the same issue, so I jumped in and tried fixing it myself, even if it was a temporarily solution. I'm linking my pull request that solves it (#241)

**Comment by Neptune650** [comment]
I see and can understand this issue, the cause is that is likely the server you're trying to connect to doesn't have that option enabled, please ask the server manager to do so.

**Comment by Neptune650** [comment]
This happens because UltimMC always uses the latest version, but the server may not.

**Comment by hiarthurbr** [comment]
As a server owner, I have disabled `enforce-secure-profile` on server.properties, and `force-key-authentication` on velocity.toml. But it seems that velocity in specific has some [kind of bug](https://github.com/PaperMC/Velocity/issues/863) that doesn't allow connections even with it disabled. The
PR linked solves that problem.

**Comment by jwaxy** [comment]
> As a server owner, I have disabled `enforce-secure-profile` on server.properties, and `force-key-authentication` on velocity.toml. But it seems that velocity in specific has some [kind of bug](https://github.com/PaperMC/Velocity/issues/863) that doesn't allow connections even with it disabled. The PR linked solves that problem.

I tried to it with PaperMC on aternos. But nothing changed :/

**Comment by alikindsys** [comment]
A workaround is having [NoChatReports](https://modrinth.com/mod/no-chat-reports/) installed on the client or server.

cc @Ar7hurz1nh0
